### DISCUSSION TITLE
Obsoleto : mb_convert_encoding(): el manejo de entidades HTML

### DISCUSSION THREAD

**Discussion by kevinlemi91** [chamilo/chamilo-lms#5849] [General]
Hola buenas noches, llevo horas intentando editar la linea que da error pero no consigo poner bien la formula necesaria. He buscado en el foro pero no encontre nada sobre el error, agradeceria que alguien me dijera como poder continuar la instalacion , ya que no me deja pasar de las casillas de los datos a rellenar los datos de la base d emi hosting. Muchas gracias:

Obsoleto : mb_convert_encoding(): el manejo de entidades HTML a través de mbstring está obsoleto; utilice htmlspecialchars, htmlentities o mb_encode_numericentity/mb_decode_numericentity en su lugar en /var/www/vhosts/41028488.xxxxxxxxxxxxxxx/chamilo/main/inc/lib/internationalization.lib.php en la línea 1210

**Comment by kevinlemi91** [comment]
Solucionado, bajando la version a 7.4 en la base desaparece dicho error. Ahi queda por si a alguien le puede ayudar

**Comment by narcisgarcia** [comment]
Incidencia relacionada con la petición de mejora https://github.com/chamilo/chamilo-lms/issues/5969

**Comment by ywarnier** [comment]
El cambio https://github.com/chamilo/chamilo-lms/commit/600d8a1a9269b21c4d5ba78f04c2a34e60fd2688 debería arreglar este problema en particular. Siguen apareciendo unos errores de tipo "DEPRECATED" en la última página del instalador, pero no impide proceder con la instalación en PHP 8.3. Ojo que todavía no hemos hecho las pruebas suficientes para garantizar que funcione integralmente Chamilo 1.11.30 con PHP 8.3, y que este cambio está en Git (rama 1.11.x) y estará oficialmente en Chamilo solo cuando salga la 1.11.30 (ojalá muy pronto).
